janusgraph-dist/src/assembly/static/conf/log4j2-console.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="warn">
+    <Appenders>
+        <Console name="STDOUT" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss} %-5level %class.%method{36} - %msg%n"/>
+        </Console>
+    </Appenders>
+    <Loggers>
+        <Root level="info">
```

Comment:
The old file `janusgraph-dist/src/assembly/static/conf/log4j-console.properties` has more configurations (e.g. `log4j.logger.org.apache.hadoop.conf.Configuration.deprecation=OFF`). Do we not need those anymore?

Reply:
log4j.logger.org.apache.hadoop.conf.Configuration.deprecation=OFF is any longer needed.

Reply:
For the other log loglevel changes, we have to see behavior in production what levels would be great. I don't really love to just copy values without knowing why. JanusGraph has a lot of stuff that is not really maintained at moment.

Reply:
My only concern is, this might change the behavior in the production, in which case we should make it clear in the changelog. However, I personally don't understand everything in the old log configuration, so I am not confident to review these configuration changes. Hope someone else could help with the review.

Reply:
I will add a note to changelog.

Reply:
Done